client/logclient.go

```diff
@@ -289,3 +289,21 @@ func (c *LogClient) GetAcceptedRoots(ctx context.Context) ([]ct.ASN1Cert, error)
 	}
 	return roots, nil
 }
+
+// GetEntryAndProof returns a log entry and audit path for the index of a leaf.
+func (c *LogClient) GetEntryAndProof(ctx context.Context, index uint64, treeSize uint64) (*ct.GetEntryAndProofResponse, error) {
```

Comment:
Could shorten to `index, treeSize uint64`.

Reply:
Done.